stl/inc/yvals_core.h

```diff
@@ -51,13 +52,16 @@
 // P0898R3 Standard Library Concepts
 // P0919R3 Heterogeneous Lookup For Unordered Containers
 // P0966R1 string::reserve() Should Not Shrink
+// P1024R3 Enhancing span Usability
+// P1085R2 Removing span Comparisons
 // P1209R0 erase_if(), erase()
 // P1227R2 Signed std::ssize(), Unsigned span::size()
-//     (partially implemented)
 // P1357R1 is_bounded_array, is_unbounded_array
+// P1394R4 Range constructor for std::span
 // P1612R1 Relocating endian To <bit>
 // P1651R0 bind_front() Should Not Unwrap reference_wrapper
 // P1754R1 Rename Concepts To standard_case
+// P1872R0 span should have size_type, not index_type

```

Comment:
Nitpick, this should be capitalized as `P1872R0 span Should Have size_type, Not index_type` as mentioned above.

Reply:
Done